components/ai_chat/resources/page/components/site_title/style.module.scss

```diff
@@ -36,9 +37,50 @@
   align-items: center;
   justify-content: center;
 
+  &:focus-visible {
+    outline: none;
+    box-shadow: var(--leo-effect-focus-state);
+  }
+
   img {
     width: 16px;
     height: 16px;
     object-fit: contain;
   }
 }
+
+.actions {
+  --leo-icon-color: var(--leo-color-text-secondary);
+}
+
+.tooltip {
+  position: absolute;
+  top: 50px;
+  z-index: 2;
+  height: fit-content;
+  max-width: 260px;
+  background-color: var(--leo-color-container-background);
+  padding: 16px;
+  box-shadow: var(--leo-effect-elevation-03);
+  border-radius: 5px;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.2s ease-in-out, visibility 0.2s ease-in-out;
```

Comment:
nit: Doesn't this feel too long to appear? Makes the UI feel a bit sluggish, to me. Maybe it should be instant and then it could still fade out?

Reply:
anything lower than 200ms is too subtle to notice. I'm not sure how to achieve just a instant fade out using css transitions. This is trivial using WAAPI.

Reply:
What do you mean by that? CSS does not have a default transition for opacity. You simply don't define a transition and when opacity changes to `1` then it will appear instantly.

Reply:
updated